lib/ctcs2_to_junit.pm

```diff
@@ -71,8 +71,8 @@ sub generateXML {
         failures  => "$fail_num",
         hostname  => `hostname`,
         id        => "0",
-        name      => get_var("QA_TESTSUITE"),
-        package   => get_var("QA_TESTSUITE"),
+        name      => get_var('QA_TESTSET') || "xfstests",
+        package   => get_var('QA_TESTSET') || "test_result",
```

Comment:
the OR can be moved inside get_var

get_var('QA_TESTSET',  "xfstests")

Reply:
I see. I'll make a quick modify for this, since it's in API description, would be OK without new VR.